examples/user/test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+cleanup() {
+  set +e
+  echo "Clean up..."
+  kubectl delete -f my-user.yaml
+  kubectl delete -f ns-per-user.yaml
+  kubectl delete configmap user-controller -n metacontroller
+}
+trap cleanup EXIT
+
+set -ex
+
+echo "Install controller..."
+kubectl create configmap user-controller -n metacontroller --from-file=sync.py
+kubectl apply -f ns-per-user.yaml
+
+echo "Create a User..."
+kubectl apply -f my-user.yaml
+
+echo "Wait for per User NS..."
+until [[ "$(kubectl get ns my-test-user -o 'jsonpath={.metadata.name}')" == "my-test-user" ]]; do sleep 1; done
```

Comment:
I think this test is a good place to check that GC works for cluster-cluster relationships, like I mentioned above for cluster-namespaced.